Cargo.toml

```diff
@@ -13,21 +13,18 @@ doctest = false
 [features]
 unstable = [
     "string_cache/unstable",
-    "tendril/unstable",
     "rc/unstable",
     "html5ever/unstable",
     "selectors/unstable",
 ]
-with-hyper = ["hyper"]
```

Comment:
Hmm. Was hyper moved to html5ever because servo needed it? Or something else. I ask because I like to keep similar interface to html5ever in xml5ever.